themes/vue/layout/partners-page.ejs

```diff
@@ -48,13 +48,13 @@
             <ul>
               <% for (let proficiency of partner.proficiencies) { %>
                 <li>
-                  <a href="<%- page.proficiencies_urls[proficiency] %>" target=_blank> <%= proficiency %></a>
+                  <a href="<%- proficiency.url || page.proficiencies_urls[proficiency] %>" target=_blank rel="noopener"> <%= proficiency.name || proficiency %></a>
                 </li>
               <% } %>
             </ul>
           </dd>
 
-          <footer class="social">
```

Comment:
Fixed a11y issue. Only `dt`, `dd`, `template`, and `script` tags are allowed under a `dl`